src/main/java/net/snowflake/client/core/SFBaseSession.java

```diff
@@ -384,6 +386,15 @@ public HttpClientSettingsKey getHttpClientKey() throws SnowflakeSQLException {
               userAgentSuffix,
               gzipDisabled);
 
+      logger.info("Driver OCSP mode: {}, gzip disabled: {}, proxy protocol: {}," +
+                  " proxy host: {}, proxy port: {}, proxy user: {}",
+              ocspMode,
+              gzipDisabled,
+              proxyProtocol,
+              proxyHost,
+              proxyPort,
+              proxyUser);
```

Comment:
should we log additional information that proxyPassword is provided (not its content)?

Reply:
Added the log. Thanks for the idea.